frontend/src/scenes/max/components/QuestionInput.tsx

```diff
@@ -163,13 +163,13 @@ export const QuestionInput = React.forwardRef<HTMLDivElement, QuestionInputProps
                 {tools.length > 0 && (
                     <div
                         className={clsx(
-                            'flex gap-1 text-xs font-medium cursor-default px-1.5',
+                            'flex flex-wrap gap-x-1 gap-y-0.5 text-xs font-medium cursor-default px-1.5 *:whitespace-nowrap',
                             !isFloating
                                 ? 'w-[calc(100%-1rem)] py-1 border-x border-b rounded-b backdrop-blur-sm bg-[var(--glass-bg-3000)]'
                                 : 'w-full -mb-1 py-0.5'
                         )}
                     >
-                        <span>Tools in context:</span>
+                        <span>Tools here:</span>
```

Comment:
"Tools" is still very technical, I would say "Available here:" and add an explanation of each tool on hover (a new arg in `<MaxTool />`), like "I can change filters on this page".

Reply:
Didn't think of it that way. I'll push back a bit here, at least for now before asking users, as our users _are_ leaning technical (even if product managers). But would be worth checking with users in an interview casually (@annikaschmid for the future)